pkg/cache/cache_test.go

```diff
@@ -11,6 +11,25 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type synonymMock struct {
+	categories map[string]string
+}
+
+func newSynonymMock() *synonymMock {
+	return &synonymMock{
+		categories: map[string]string{
+			"go":           "golang",
+			"golang":       "go",
+			"cryptography": "криптография",
+			"криптография": "cryptography",
+		},
+	}
+}
+
+func (s *synonymMock) GetSynonymCategory(category string) string {
```

Comment:
your actual implementation is doing the same, you don't need interface here
stop complicating things!!!